2021-05-18 12:12:48,522 INFO: Microblog startup [in C:\Users\ÀÌÁöÈÆ\Desktop\miguel\microblog\app\__init__.py:47]
2021-05-18 12:18:40,002 INFO: Microblog startup [in C:\Users\ÀÌÁöÈÆ\Desktop\miguel\microblog\app\__init__.py:47]
2021-05-18 12:19:54,317 INFO: Microblog startup [in C:\Users\ÀÌÁöÈÆ\Desktop\miguel\microblog\app\__init__.py:47]
2021-05-18 12:19:57,863 ERROR: Exception on /login [GET] [in c:\users\ÀÌÁöÈÆ\desktop\miguel\microblog\venv\lib\site-packages\flask\app.py:1891]
Traceback (most recent call last):
  File "c:\users\ÀÌÁöÈÆ\desktop\miguel\microblog\venv\lib\site-packages\flask\app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\users\ÀÌÁöÈÆ\desktop\miguel\microblog\venv\lib\site-packages\flask\app.py", line 1953, in full_dispatch_request
    return self.finalize_request(rv)
  File "c:\users\ÀÌÁöÈÆ\desktop\miguel\microblog\venv\lib\site-packages\flask\app.py", line 1968, in finalize_request
    response = self.make_response(rv)
  File "c:\users\ÀÌÁöÈÆ\desktop\miguel\microblog\venv\lib\site-packages\flask\app.py", line 2097, in make_response
    raise TypeError(
TypeError: The view function did not return a valid response. The function either returned None or ended without a return statement.
2021-05-18 12:20:47,746 INFO: Microblog startup [in C:\Users\ÀÌÁöÈÆ\Desktop\miguel\microblog\app\__init__.py:47]
2021-05-18 12:20:49,961 ERROR: Exception on /login [GET] [in c:\users\ÀÌÁöÈÆ\desktop\miguel\microblog\venv\lib\site-packages\flask\app.py:1891]
Traceback (most recent call last):
  File "c:\users\ÀÌÁöÈÆ\desktop\miguel\microblog\venv\lib\site-packages\flask\app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\users\ÀÌÁöÈÆ\desktop\miguel\microblog\venv\lib\site-packages\flask\app.py", line 1953, in full_dispatch_request
    return self.finalize_request(rv)
  File "c:\users\ÀÌÁöÈÆ\desktop\miguel\microblog\venv\lib\site-packages\flask\app.py", line 1968, in finalize_request
    response = self.make_response(rv)
  File "c:\users\ÀÌÁöÈÆ\desktop\miguel\microblog\venv\lib\site-packages\flask\app.py", line 2097, in make_response
    raise TypeError(
TypeError: The view function did not return a valid response. The function either returned None or ended without a return statement.
2021-05-18 12:21:38,449 INFO: Microblog startup [in C:\Users\ÀÌÁöÈÆ\Desktop\miguel\microblog\app\__init__.py:47]
2021-05-18 12:21:41,852 ERROR: Exception on /login [GET] [in c:\users\ÀÌÁöÈÆ\desktop\miguel\microblog\venv\lib\site-packages\flask\app.py:1891]
Traceback (most recent call last):
  File "c:\users\ÀÌÁöÈÆ\desktop\miguel\microblog\venv\lib\site-packages\flask\app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\users\ÀÌÁöÈÆ\desktop\miguel\microblog\venv\lib\site-packages\flask\app.py", line 1953, in full_dispatch_request
    return self.finalize_request(rv)
  File "c:\users\ÀÌÁöÈÆ\desktop\miguel\microblog\venv\lib\site-packages\flask\app.py", line 1968, in finalize_request
    response = self.make_response(rv)
  File "c:\users\ÀÌÁöÈÆ\desktop\miguel\microblog\venv\lib\site-packages\flask\app.py", line 2097, in make_response
    raise TypeError(
TypeError: The view function did not return a valid response. The function either returned None or ended without a return statement.
2021-05-18 12:21:48,155 ERROR: Exception on /login [GET] [in c:\users\ÀÌÁöÈÆ\desktop\miguel\microblog\venv\lib\site-packages\flask\app.py:1891]
Traceback (most recent call last):
  File "c:\users\ÀÌÁöÈÆ\desktop\miguel\microblog\venv\lib\site-packages\flask\app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\users\ÀÌÁöÈÆ\desktop\miguel\microblog\venv\lib\site-packages\flask\app.py", line 1953, in full_dispatch_request
    return self.finalize_request(rv)
  File "c:\users\ÀÌÁöÈÆ\desktop\miguel\microblog\venv\lib\site-packages\flask\app.py", line 1968, in finalize_request
    response = self.make_response(rv)
  File "c:\users\ÀÌÁöÈÆ\desktop\miguel\microblog\venv\lib\site-packages\flask\app.py", line 2097, in make_response
    raise TypeError(
TypeError: The view function did not return a valid response. The function either returned None or ended without a return statement.
2021-05-18 12:22:13,726 ERROR: Exception on /login [GET] [in c:\users\ÀÌÁöÈÆ\desktop\miguel\microblog\venv\lib\site-packages\flask\app.py:1891]
Traceback (most recent call last):
  File "c:\users\ÀÌÁöÈÆ\desktop\miguel\microblog\venv\lib\site-packages\flask\app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\users\ÀÌÁöÈÆ\desktop\miguel\microblog\venv\lib\site-packages\flask\app.py", line 1953, in full_dispatch_request
    return self.finalize_request(rv)
  File "c:\users\ÀÌÁöÈÆ\desktop\miguel\microblog\venv\lib\site-packages\flask\app.py", line 1968, in finalize_request
    response = self.make_response(rv)
  File "c:\users\ÀÌÁöÈÆ\desktop\miguel\microblog\venv\lib\site-packages\flask\app.py", line 2097, in make_response
    raise TypeError(
TypeError: The view function did not return a valid response. The function either returned None or ended without a return statement.
2021-05-18 12:22:15,764 ERROR: Exception on /login [GET] [in c:\users\ÀÌÁöÈÆ\desktop\miguel\microblog\venv\lib\site-packages\flask\app.py:1891]
Traceback (most recent call last):
  File "c:\users\ÀÌÁöÈÆ\desktop\miguel\microblog\venv\lib\site-packages\flask\app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\users\ÀÌÁöÈÆ\desktop\miguel\microblog\venv\lib\site-packages\flask\app.py", line 1953, in full_dispatch_request
    return self.finalize_request(rv)
  File "c:\users\ÀÌÁöÈÆ\desktop\miguel\microblog\venv\lib\site-packages\flask\app.py", line 1968, in finalize_request
    response = self.make_response(rv)
  File "c:\users\ÀÌÁöÈÆ\desktop\miguel\microblog\venv\lib\site-packages\flask\app.py", line 2097, in make_response
    raise TypeError(
TypeError: The view function did not return a valid response. The function either returned None or ended without a return statement.
2021-05-18 12:22:17,150 ERROR: Exception on /login [GET] [in c:\users\ÀÌÁöÈÆ\desktop\miguel\microblog\venv\lib\site-packages\flask\app.py:1891]
Traceback (most recent call last):
  File "c:\users\ÀÌÁöÈÆ\desktop\miguel\microblog\venv\lib\site-packages\flask\app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\users\ÀÌÁöÈÆ\desktop\miguel\microblog\venv\lib\site-packages\flask\app.py", line 1953, in full_dispatch_request
    return self.finalize_request(rv)
  File "c:\users\ÀÌÁöÈÆ\desktop\miguel\microblog\venv\lib\site-packages\flask\app.py", line 1968, in finalize_request
    response = self.make_response(rv)
  File "c:\users\ÀÌÁöÈÆ\desktop\miguel\microblog\venv\lib\site-packages\flask\app.py", line 2097, in make_response
    raise TypeError(
TypeError: The view function did not return a valid response. The function either returned None or ended without a return statement.
2021-05-18 12:24:47,974 INFO: Microblog startup [in C:\Users\ÀÌÁöÈÆ\Desktop\miguel\microblog\app\__init__.py:47]
2021-05-18 12:24:50,741 ERROR: Exception on /login [GET] [in c:\users\ÀÌÁöÈÆ\desktop\miguel\microblog\venv\lib\site-packages\flask\app.py:1891]
Traceback (most recent call last):
  File "c:\users\ÀÌÁöÈÆ\desktop\miguel\microblog\venv\lib\site-packages\flask\app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\users\ÀÌÁöÈÆ\desktop\miguel\microblog\venv\lib\site-packages\flask\app.py", line 1953, in full_dispatch_request
    return self.finalize_request(rv)
  File "c:\users\ÀÌÁöÈÆ\desktop\miguel\microblog\venv\lib\site-packages\flask\app.py", line 1968, in finalize_request
    response = self.make_response(rv)
  File "c:\users\ÀÌÁöÈÆ\desktop\miguel\microblog\venv\lib\site-packages\flask\app.py", line 2097, in make_response
    raise TypeError(
TypeError: The view function did not return a valid response. The function either returned None or ended without a return statement.
2021-05-18 12:25:04,112 INFO: Microblog startup [in C:\Users\ÀÌÁöÈÆ\Desktop\miguel\microblog\app\__init__.py:47]
2021-05-18 12:25:05,485 ERROR: Exception on /login [GET] [in c:\users\ÀÌÁöÈÆ\desktop\miguel\microblog\venv\lib\site-packages\flask\app.py:1891]
Traceback (most recent call last):
  File "c:\users\ÀÌÁöÈÆ\desktop\miguel\microblog\venv\lib\site-packages\flask\app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\users\ÀÌÁöÈÆ\desktop\miguel\microblog\venv\lib\site-packages\flask\app.py", line 1953, in full_dispatch_request
    return self.finalize_request(rv)
  File "c:\users\ÀÌÁöÈÆ\desktop\miguel\microblog\venv\lib\site-packages\flask\app.py", line 1968, in finalize_request
    response = self.make_response(rv)
  File "c:\users\ÀÌÁöÈÆ\desktop\miguel\microblog\venv\lib\site-packages\flask\app.py", line 2097, in make_response
    raise TypeError(
TypeError: The view function did not return a valid response. The function either returned None or ended without a return statement.
2021-05-18 12:25:37,822 INFO: Microblog startup [in C:\Users\ÀÌÁöÈÆ\Desktop\miguel\microblog\app\__init__.py:47]
2021-05-18 12:25:39,885 ERROR: Exception on /login [GET] [in c:\users\ÀÌÁöÈÆ\desktop\miguel\microblog\venv\lib\site-packages\flask\app.py:1891]
Traceback (most recent call last):
  File "c:\users\ÀÌÁöÈÆ\desktop\miguel\microblog\venv\lib\site-packages\flask\app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\users\ÀÌÁöÈÆ\desktop\miguel\microblog\venv\lib\site-packages\flask\app.py", line 1953, in full_dispatch_request
    return self.finalize_request(rv)
  File "c:\users\ÀÌÁöÈÆ\desktop\miguel\microblog\venv\lib\site-packages\flask\app.py", line 1968, in finalize_request
    response = self.make_response(rv)
  File "c:\users\ÀÌÁöÈÆ\desktop\miguel\microblog\venv\lib\site-packages\flask\app.py", line 2097, in make_response
    raise TypeError(
TypeError: The view function did not return a valid response. The function either returned None or ended without a return statement.
